ZsensorBH1750.ino

```diff
@@ -0,0 +1,124 @@
+/*  
+  OpenMQTTGateway Addon  - ESP8266 or Arduino program for home automation 
+
+   Act as a wifi or ethernet gateway between your 433mhz/infrared IR signal  and a MQTT broker 
+   Send and receiving command by MQTT
+ 
+  Light Meter Addon
+
+  Measures ambient Light Intensity in Lux (lx), Foot Candela (fc) and Watt/m^2 (wqm)
+
+  Required Hardware Module: BH1750
+
+
+  Connection Schemata:
+  --------------------
+
+  BH1750 ------> Arduino Uno ----------> ESP8266
+  Vcc ---------> 5V -------------------> Vu (5V)
```

Comment:
A VCC of 5V is out of spec for both (bh1750 & esp8266) and will damage the components. Please supply 3.3V.